lib/vagrant-libvirt/config.rb

```diff
@@ -279,26 +279,26 @@ def _get_cdrom_dev(cdroms)
       end
 
```

Comment:
@electrofelix can you help me with understanding the merge method? Reach me on gitter if that is easier.